library-best-practices/JLBP-1.md

```diff
@@ -9,7 +9,7 @@ that library and that library's consumers.
 
 Some specific notes about minimizing dependencies:
 
-- Use the smallest scope possible. For example, auto-value doesn't
+- Use the smallest scope possible. For example, `auto-value` doesn't
```

Comment:
I think just AutoValue in no quotes is fine. Either that or give the complete coordinates

Reply:
Updated to AutoValue